### PR TITLE
DOC: Correct get_state doc

### PR DESCRIPTION
**Pull request by bashtage**
Move flag to parameters

### CODE REVIEW

File: numpy/random/mtrand.pyx [hunk 1]
```diff
@@ -266,10 +272,6 @@ cdef class RandomState:
             If `legacy` is False, or the BitGenerator is not NT19937, then
```
Comment (eric-wieser) [inline numpy/random/mtrand.pyx]:
```suggestion
            If `legacy` is False, or the BitGenerator is not MT19937, then
```

File: numpy/random/mtrand.pyx [hunk 2]
```diff
@@ -252,6 +252,12 @@ cdef class RandomState:
 
         For more details, see `set_state`.
 
+        Parameters
+        ----------
+        legacy : bool, optional
+            Flag indicating the return a legacy tuple state when the BitGenerator
```
Comment (eric-wieser) [inline numpy/random/mtrand.pyx]:
```suggestion
            Flag indicating to return a legacy tuple state when the BitGenerator
```

File: numpy/random/mtrand.pyx [hunk 3]
```diff
@@ -252,6 +252,12 @@ cdef class RandomState:
...
+            is MT19937.
```
Comment (eric-wieser) [inline numpy/random/mtrand.pyx]:
```suggestion
            is MT19937, instead of a ``dict``.
```

File: numpy/random/mtrand.pyx (hunk 1 above)
Comment (bashtage) [reply numpy/random/mtrand.pyx]: It is spaced with 12, which I think is right.

File: numpy/random/mtrand.pyx (hunk 3 above)
Comment (bashtage) [reply numpy/random/mtrand.pyx]: Fixed.

File: numpy/random/mtrand.pyx (hunk 2 above)
Comment (bashtage) [reply numpy/random/mtrand.pyx]: Fixed.

File: numpy/random/mtrand.pyx (hunk 1 above)
Comment (bashtage) [reply numpy/random/mtrand.pyx]: Fixed N->M, but leading space seems right at 12.

Comment (eric-wieser) [reply numpy/random/mtrand.pyx]: Leading space is a GitHub bug, hitting "commit suggestion" doesn't actually add it.

**Review by eric-wieser — APPROVED** [review]

**Comment by mattip** [comment]
Thanks @bashtage

**Pull request merged**